app/src/main/java/com/example/cm/data/map/SnapPagerScrollListener.java

```diff
@@ -0,0 +1,91 @@
+package com.example.cm.data.map;
+
+import android.view.View;
+
+import androidx.annotation.IntDef;
+import androidx.annotation.NonNull;
+import androidx.recyclerview.widget.PagerSnapHelper;
+import androidx.recyclerview.widget.RecyclerView;
+
+
+/**
+ * From: https://stackoverflow.com/questions/52083768/from-to-page-changed-listener-in-pagersnaphelper
+ * Used to get position of the current item in the RecyclerView when snapped
+ */
+public class SnapPagerScrollListener extends RecyclerView.OnScrollListener {
+
+    // Constants
+    public static final int ON_SCROLL = 0;
```

Comment:
lieber in Constants?

Reply:
Keeping it here since it's only really used in this corresponding file